### DISCUSSION TITLE
Sending alerts on a prometheus pushgateway

### DISCUSSION THREAD

**Discussion by tony059** [jertel/elastalert2#1475] [Q&A] [answered]
It's all in the title... can you tell me if we have the possibility to send alerts on a prometheus pushgateway with Elastalert2 ?

Thanks in advance.

**Comment by jertel** [comment, marked as answer]
ElastAlert2 only provides the built-in prometheus server. It may be possible to setup a companion pusher that scrapes the server and then pushes the metrics outward, but nothing about that would be specific to ElastAlert 2.
